gulpfile.js

```diff
@@ -187,6 +187,10 @@ gulp.task('watch', function() {
   gulp.watch(paths.src, ['modules']);
 });
 
+gulp.task('dev', function() {
```

Comment:
Agreed, will remove this on the Tech review amend commit, in the meantime already created a PR for it https://github.com/facebook/draft-js/pull/242